uw-frame-components/portal/misc/partials/launch-button.html

```diff
@@ -0,0 +1,5 @@
+<a class="launch-app-button ng-scope"
+   target="{{ target }}"
+   ng-href="{{ href }}">
```

Comment:
Should this directive accept an additional parameter that provides say [an `aria-label`](https://www.w3.org/TR/WCAG20-TECHS/ARIA8.html)?  This could encourage implementing [design guidance](https://docs.google.com/document/d/1JQOe0a9XPZdTev-M7qyl_bRHn3acBSVJLkTvx57uIf4/edit) to e.g. supplement a "Launch app" visual text label with a ARIA attributes that clarify what app will be launched ("This button will launch the Time and Absence app in MyUW.").